management-api-agent-common/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -830,17 +832,16 @@ public String repair(
     }
     // set incremental reapir
     repairSpec.put(RepairOption.INCREMENTAL_KEY, Boolean.toString(!full));
+
+    // set repair parallelism
+    repairSpec.put(RepairOption.PARALLELISM_KEY, repairParallelism);
```

Comment:
issue: Isn't this overwritten by the write on L842?

Reply:
it is if we're running an incremental repair (`!full`).
That's the behavior I want:
we use the provided parallelism, unless it's an incremental repair, then we force to parallel.